### PR TITLE
Infoblox cannot deal with broken DNS data due to long message

### PR DESCRIPTION
**Pull request by philippkahr**
When dealing with very long messages Infoblox at some point ends the message with a `...` symbol.

Example: `cool.server.production.elastic.co. 190 IN A 175.16.199.31; cool.server.production.elastic.co ...`

However, our pipeline expects it is always in `cool.server.production.elastic.co. 190 IN A 175.16.199.31` format for the script to work since it splits this data based on a space symbol and then assigns the individual values such to the respective fields.

However with `elastic.co ...` it does not work anymore and it will throw `index 2 out of bound` error. 

* I fixed the script to only perform the field mapping if the array is length greater or equal to 4 values.
* I fixed a 2nd issue. The network.transport sometimes was extracted with the `view 12: TCP` value. I added a dissect to extract just the `TCP` field and lowercase it. I could have manipulated the original groks, but they are already complicated and I feel like this dissect is a better fit.

I am on PTO from 4 November till 20 November.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/vsphere/data_stream/log/_dev/test/pipeline/test-format-common.log
Comment (efd6) [inline packages/vsphere/data_stream/log/_dev/test/pipeline/test-format-common.log:1]: I suspect this change should not be here.

Comment (philippkahr) [reply packages/vsphere/data_stream/log/_dev/test/pipeline/test-format-common.log:1]: Correct 🙈 , let me get it out of this PR

File: packages/infoblox_nios/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -136,6 +136,13 @@ processors:
       target_field: client.as.organization.name
       ignore_missing: true
       if: ctx.client?.as?.organization_name != null
+  - dissect:
+      field: network.transport
+      pattern: "view %{}: %{network.transport}"
+      if: "ctx.network?.transport?.contains('view') ?: false"
```
Comment (efd6) [inline packages/infoblox_nios/data_stream/log/elasticsearch/ingest_pipeline/default.yml:142]:
```suggestion
      if: ctx.network?.transport instanceof String && ctx.network.transport.contains('view')
```

**Comment by mrodm** [comment]
Hi @philippkahr, please update your branch with the latest contents from main branch. There was an important PR merged updating the CI pipelines. Thanks!

**Review by efd6 — APPROVED** [review]
LGTM. The use of arrays for the answers.* doesn't match the definitions for the fields in ECS, but it is the only sensible way to do it and we already use array for this here and in other integrations.

**Pull request merged**